Tests/SPMTest/SPMTest/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AppDelegate.swift
+//  SPMTest
+//
+//  Created by Alexander Azarov on 13.04.21.
+//
```

Comment:
Nit: This repository relies on a LICENSE.md in the root folder, so a comment block is unnecessary.

Reply:
Xcode created this automatically and I forgot to delete, thx for pointing out.